extensions/youtube/package.json

```diff
@@ -31,15 +31,15 @@
   ],
   "preferences": [
     {
-      "name": "apikey",
+      "name": "apiKey",
```

Comment:
Can we avoid changing this? This will do so everyone will lose the current API key